src/aap_eda/core/tasking/__init__.py

```diff
@@ -158,7 +159,7 @@ def __init__(
         connection: Optional[Connection] = None,
         exc_handler: Any = None,
         exception_handlers: _ErrorHandlersArgType = None,
-        default_worker_ttl: int = DEFAULT_WORKER_TTL,
+        default_worker_ttl: int = 10,
```

Comment:
@Alex-Izquierdo Should this be using settings.SOMETHING?

Reply:
Good catch. rolling back to the default one. It is overrided later in line 185 anyway. 
https://github.com/ansible/eda-server/pull/701/files#diff-8919c4e8183e21876fcf7b5ad8884b1efbb37a0e08544fba092d83c6f733adfbR185